assets/scss/components/_card.scss

```diff
@@ -59,3 +59,13 @@
     flex-direction: column;
     justify-content: space-between;
 }
+
+.c-card__price {
+    display: flex;
+    gap: 0.75rem;
+    align-items: center;
+}
+
+.c-card__rating {
```

Comment:
@mgbelegu instead of this one, I'd suggest creating a utility class like `.u-no-margin`